pkg/core/blockchain.go

```diff
@@ -628,6 +629,122 @@ func (bc *Blockchain) jumpToStateInternal(p uint32, stage stateJumpStage) error
 	return nil
 }
 
+// Reset resets chain state to the specified height if possible.
```

Comment:
It can't be called on a running blockchain which is important.

Reply:
Added a comment about that.